homepage/settings.html

```diff
@@ -106,9 +106,125 @@ <h1 class="display-3 text-center">Settings</h1>
           </div>
         </div>
       </div>
-
     </div>
+    <div class="container">
+      <div class="row justify-content-center">
+        <div class="col-12 col-lg-10 col-xl-8 mx-auto">
+          <div class="my-4">
+            <ul class="nav nav-tabs mb-4" id="myTab" role="tablist">
+              <li class="nav-item">
+                <a class="nav-link active" id="contact-tab" data-toggle="tab" href="#contact" role="tab" aria-controls="contact"
+                  aria-selected="false">Username and Password</a>
+              </li>
+            </ul>
+            <div class="list-group mb-5 shadow">
+              <div class="list-group-item">
+                <div class="row align-items-center">
+                  <div class="col">
+                    <strong class="mb-0">Reset Username</strong>
+                    <form>
+                      <div class="form-group">
+                        <input type="email" class="form-control" id="exampleInputEmail1" aria-describedby="emailHelp" placeholder="Enter your new username">
+                        <br>
+                        <button type="button" class="btn btn-secondary btn-sm">Confirm Changes</button>
```

Comment:
should be setting a new "display name" because we're not planning to allow username changes (although we probably can, the idea is that the display name is what you see/change)